documentation/commands/git-new-pull-request.md

```diff
@@ -12,10 +12,12 @@ git new-pull-request
 
 #### DESCRIPTION
 
-Opens a browser window to the form for a new pull request page
-for the current branch on the code hosting platform.
+Syncs the current branch
+and opens a browser window to the new pull request page of your repository.
 
-The pull request only shows the changes made against the immediate parent branch.
+The form is pre-populated for the current branch
+so that the pull request only shows the changes made
+against the immediate parent branch.
```

Comment:
These changes are unrelated to this PR. Looks like they were meant for another branch